tests/e2e-tests/specs/front-end/my-account.test.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * @format
+ */
+
+/**
+ * External dependencies
+ */
+import { switchUserToTest } from '@wordpress/e2e-test-utils';
+
+const customerUsername = process.env.WP_USERNAME;
+
+describe( 'My account page', () => {
+	beforeAll( async () => {
+		await switchUserToTest();
+	} );
+
+	it( 'allows customer to login', async () => {
+		await expect( page ).toMatch( 'Hello ' + customerUsername );
```

Comment:
This only works if customer does not have the _first_ and _last name_ filled in, otherwise it uses those instead. 

<img width="797" alt="Screenshot 2019-10-15 at 18 16 10" src="https://user-images.githubusercontent.com/2207451/66849878-ff9aea00-ef77-11e9-8a77-4a1fff7e2c60.png">

Reply:
@peterfabian thanks for the review and for catching the use case I didn't cover! I see that it fails indeed if the customer has the first and last name filled in wp-admin. The word `Hello` is still there though. 

I don't think we can explicitly verify the first and last name unless we pass it to the test to check. I removed the username verification and left just the `Hello` word. After all, as long as we can login with customer's username and password, we can be confident enough that we are logged in as a correct user. I did add more verification conditions such as make sure the `Downloads`, `Addresses`, `Account`, `Logout` fields are there. 

Please review again and let me know your thoughts.